frontend/src/app/signUp/Organization/OrganizationForm.tsx

```diff
@@ -179,17 +179,16 @@ const OrganizationForm = () => {
           return (
             <>
               <label className="usa-label margin-top-3">
-                <strong>{label}</strong>
-                <TextWithTooltip
-                  tooltip="Can be anyone that works at your organization. Adds testing facility locations; manages the account and user access."
-                  text="What's an organization administrator?"
-                />
+                <strong>
+                  <TextWithTooltip
+                    tooltip="Can be anyone that works at your organization. Adds testing facility locations; manages the account and user access."
+                    text="Organization administrator"
+                  />
+                </strong>
               </label>
               <p className="usa-hint">
                 Only one person from an organization can be the administrator.
-                This person will submit personal information for identity
-                verification. (SimpleReport doesn't access or keep personal
-                identity information.)
+                This person will submit information for identity verifcation.
```

Comment:
I think we want to keep the "SimpleReport doesn't access or keep personal identity information" piece, right @Rebecca-LM?

Reply:
@emmastephenson correct.

Reply:
Okay, I wasn't sure about this since it wasn't included on the Figma, I'll add it back